kirby/config/blocks/markdown/markdown.php

```diff
@@ -1,2 +1,2 @@
 <?php /** @var \Kirby\Cms\Block $block */ ?>
-<?= $block->text()->kt();
+<?= $block->text()->html()->kt();
```

Comment:
I'm not sure. But this does not damage markdown tags but escape HTML tags successfully.

Reply:
There's an issue with `<http://link.com>` and mixed HTML in markdown.

Reply:
So let's revert

Reply:
Reverted